tests/fixtures/events.json

```diff
@@ -5,5 +5,18 @@
    "invalidEvent1": {},
    "invalidEvent2": { "en":{"name": "Dinner at Alreda"}, "categories": ["eating"] },
    "invalidEvent3": { "en":{"name": "Dinner at Alreda"}, "categories": [null, "dining"] },
-   "invalidEvent4": { "en":{"name": "Dinner at Alreda"}, "categories": []}
+   "invalidEvent4": { "en":{"name": "Dinner at Alreda"}, "categories": []},
+
+   "event": {
```

Comment:
why is this called `event`. The ones above are called `validEvent`

Reply:
removed